src/pages/administration/students/index.vue

```diff
@@ -436,4 +479,33 @@ a.action-button {
 		}
 	}
 }
+span {
+	font-weight: var(--font-weight-normal);
+}
+.content {
```

Comment:
On smaller devices the text is hard to read, probably something like this would correct it? 
```suggestion
.content {
	max-height: 30vh;
	overflow-y: scroll;
```
![Screenshot from 2020-05-06 16-23-49](https://user-images.githubusercontent.com/41312826/81188768-0a6da880-8fb6-11ea-938b-d7571a208b12.png)